jena-fuseki2/jena-fuseki-geosparql/pom.xml

```diff
@@ -98,6 +98,17 @@
           <artifactId>log4j-slf4j-impl</artifactId>
         </dependency>
 
+        <!-- Put in to ensure the dependency isn't optional. -->
+        <dependency>
+          <groupId>org.apache.logging.log4j</groupId>
+          <artifactId>log4j-api</artifactId>
+        </dependency>
+
+        <dependency>
+          <groupId>org.apache.logging.log4j</groupId>
+          <artifactId>log4j-core</artifactId>
```

Comment:
I think `<scope>runtime</scope>` can be added to this dependency and other `-impl` dependencies as logging code should only directly depend on the `api`.

Reply:
These are for the most-part packaging classes.

jena-fuseki-geosparql is both packing and code for the command line and Fuseki configuration.  Might be better to leave it as scope-compile but jena-fuseki-server should be scope-runtime as it's only shade-packaging.

Reply:
(pushed and change for scope-setting for jena-fuseki-server)